lib/services/internalEngine/index.js

```diff
@@ -509,8 +509,8 @@ class InternalEngine {
    *
    * @returns {Promise}
    */
-  getMapping (data) {
-    return this.esWrapper.getMapping(data);
+  getMapping (data, includeMeta = false) {
```

Comment:
We should stop calling Kuzzle Info "meta". We should call them Kuzzle Info.
```suggestion
  getMapping (data, includeKuzzleInfo = false) {
```